spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -402,5 +402,79 @@
         end
       end
     end
+
+    describe "line 15 wages" do
```

Comment:
[SAND] Is it reasonable to try a test case where we have an overflow? e.g. W2 wages sum to 10000000000.00? I understand that someone earning that much would probably be screened out of our tool but I'm curious how our application would handle a case like that.

Reply:
Nope, no need! You can see a conversation about this question in the story, and also more context [in this slack thread](https://njcio.slack.com/archives/C07GYAQ4E1E/p1726515777378459)